### PR TITLE
Add `Capabilities` to `Account` create and update

### PR DESCRIPTION
**Pull request by cjavilla-stripe**
Add `capabilities` to `Account`

r? @remi-stripe 
cc @stripe/api-libraries

### CODE REVIEW

File: src/Stripe.net/Services/Accounts/AccountCapabilitiesAuBecsDebitPaymentsOptions.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class AccountCapabilitiesAuBecsDebitPaymentsOptions : INestedOptions
+    {
+        /// <summary>
+        /// Passing true requests the capability for the account, if it is not
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Accounts/AccountCapabilitiesAuBecsDebitPaymentsOptions.cs:8]: This should be `<c>true</c>` instead but that will mess your indentation for all comments 😓

File: src/Stripe.net/Services/Accounts/AccountCapabilitiesAuBecsDebitPaymentsOptions.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        /// already requested. A requested capability may not immediately
+        /// become active. Any requirements to activate the capability are
+        /// returned in the requirements arrays.
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Accounts/AccountCapabilitiesAuBecsDebitPaymentsOptions.cs:11]: Same here we in theory should use CRef for `requirements`.

File: src/Stripe.net/Services/Accounts/AccountCapabilitiesOptions.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class AccountCapabilitiesOptions : INestedOptions
+    {
+        /// <summary>
+        /// The au_becs_debit_payments capability.
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Accounts/AccountCapabilitiesOptions.cs:8]: We don't use snake case in docstrings in dotnet usually and would write `The AU BECS Debit payments capability` usually

File: src/Stripe.net/Services/Accounts/AccountCreateOptions.cs [hunk 4]
```diff
@@ -25,6 +25,18 @@ public class AccountCreateOptions : BaseOptions, IHasMetadata
...
+        /// and each capability maps to its settings (e.g. whether
+        /// it has been requested or not). Each capability will be
+        /// inactive until you have provided its specific
+        /// requirements and Stripe has verified them. An account
+        /// may have some of its requested capabilities be active
+        /// and some be inactive.
+        /// </summary>
+        [JsonProperty("capabilities")]
+        public AccountCapabilitiesOptions Capabilities { get; set; }
+
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Accounts/AccountCreateOptions.cs:39]: In theory you should make `RequestedCapabilities` obsolete now

File: src/Stripe.net/Services/Accounts/AccountUpdateOptions.cs [hunk 5]
```diff
@@ -25,6 +25,18 @@ public class AccountUpdateOptions : BaseOptions, IHasMetadata
...
+        /// it has been requested or not). Each capability will be
+        /// inactive until you have provided its specific
+        /// requirements and Stripe has verified them. An account
+        /// may have some of its requested capabilities be active
+        /// and some be inactive.
+        /// </summary>
+        [JsonProperty("capabilities")]
+        public AccountCapabilitiesOptions Capabilities { get; set; }
+
         /// <summary>
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Accounts/AccountUpdateOptions.cs:40]: In theory you should make `RequestedCapabilities` obsolete now

File: src/Stripe.net/Services/Accounts/AccountCreateOptions.cs (hunk 4 above)
Comment (cjavilla-stripe) [reply src/Stripe.net/Services/Accounts/AccountCreateOptions.cs:40]: done

File: src/Stripe.net/Services/Accounts/AccountUpdateOptions.cs (hunk 5 above)
Comment (cjavilla-stripe) [reply src/Stripe.net/Services/Accounts/AccountUpdateOptions.cs:41]: done

**Pull request merged**